src/qibosoq/rfsoc_server.py

```diff
@@ -64,7 +59,7 @@ def execute_program(data: dict, qick_soc: QickSoc) -> dict:
     )
 
     asm_prog = program.asm()
-    qick_logger.handlers[0].doRollover()
+    qick_logger.handlers[0].doRollover()  # type: ignore
```

Comment:
To be checked (but Qick once more)

Reply:
No this is not qick, is something related to the loggers that I define. 
`qick_logger.handlers` is typed as a list of `Handler`, but `doRollover` is a function just of `RotatingHandler`

Reply:
Ah, you're right. I read `qick`, but I have been superficial. Where is that `.handlers` gets its `list[Handler]` type? Standard library should be properly typed since a while...

Reply:
Actually, I think it's mypy that infers that typing. In the logging module it seems non typed:
https://github.com/python/cpython/blob/a6313d78f21f79ca64dedd38e637509dc530a1b6/Lib/logging/__init__.py#L1456

Reply:
https://github.com/python/typeshed/blob/main/stdlib/logging/handlers.pyi

Reply:
So here it is! https://github.com/python/typeshed/blob/754473a30d50b918ac6f1a824f41600806defa94/stdlib/logging/__init__.pyi#L105